estimagic/optimization/consolidate_constraints.py

```diff
@@ -163,7 +163,7 @@ def _consolidate_fixes_with_equality_constraints(fixed_pc, equality_pc, params):
             assert (
                 len(valcounts) == 1
             ), "Equality constrained parameters cannot be fixed to different values."
-            fixed_value.iloc[eq["index"]] = valcounts.idex[0]
```

Comment:
Ups. Sorry, I have overlooked this gem in the last PR, right?🤦‍♂

Reply:
Yes ;-)